opslevel/resource_opslevel_service.go

```diff
@@ -62,10 +62,18 @@ func newServiceResourceModel(ctx context.Context, service opslevel.Service, owne
 		Language:        OptionalStringValue(service.Language),
 		LifecycleAlias:  OptionalStringValue(service.Lifecycle.Alias),
 		Name:            RequiredStringValue(service.Name),
-		Owner:           OptionalStringValue(ownerIdentifier),
 		Product:         OptionalStringValue(service.Product),
 		TierAlias:       OptionalStringValue(service.Tier.Alias),
 	}
+	// set owner team
```

Comment:
Should have originally included this check in https://github.com/OpsLevel/terraform-provider-opslevel/pull/333

Reply:
would have preferred for this to be split out to a separate PR.  Lets make sure we do this next time.